plat/xilinx/zynqmp/pm_service/pm_svc_main.c

```diff
@@ -54,7 +54,7 @@ int pm_setup(void)
 	if (!zynqmp_is_pmu_up())
 		return -ENODEV;
 
-	status = pm_ipi_init();
+	status = pm_ipi_init(primary_proc);
```

Comment:
At line 59, you could change to status >= 0 and then remove the pm_ipi_init that sets ret = 0 when >0.

Reply:
This comment has been addressed with the changes in the following lines